### PR TITLE
Helper: add support for cockpit widgets

### PR DESCRIPTION
**Pull request by Williangalvani**
related to https://github.com/bluerobotics/ping-viewer-next/pull/46
and https://github.com/bluerobotics/cockpit/pull/1499

### CODE REVIEW

File: core/services/helper/main.py [hunk 1]
```diff
@@ -93,6 +93,11 @@ class WebsiteStatus(BaseModel):
     error: Optional[str] = None
 
 
+class CockpitWidget(BaseModel):
+    name: str
+    url: str
+
```
Comment (joaoantoniocardoso) [inline core/services/helper/main.py:99]: Thinking on compatibility, it should include a widget version, and a cockpit target version as well

Comment (Williangalvani) [reply core/services/helper/main.py:99]: yeah, this is the absolutely minimum. I'm thinking of how to expand things without messing up things too much.
would it be an issue if we we had a complex json in here?
should we instead just use this as an aggregator and have it link to another json on the extension?
then we could also manually input that json into cockpit

**Comment by Williangalvani** [comment]
I added more info about it at https://github.com/bluerobotics/cockpit/pull/1499

**Review by joaoantoniocardoso — APPROVED** [review]

File: core/services/helper/main.py [hunk 2]
```diff
@@ -107,6 +107,7 @@ class ServiceMetadata(BaseModel):
     api: str
     sanitized_name: Optional[str]
     works_in_relative_paths: Optional[bool]
+    extras: Optional[Dict[str, str]]
```
Comment (joaoantoniocardoso) [inline core/services/helper/main.py:110]: maybe we should name it `integrations`

Comment (Williangalvani) [reply core/services/helper/main.py:110]: I think that limits the usage. I don't want to tell people what they can or can't add in these jsons. not sure if that makes sense.

Comment (joaoantoniocardoso) [reply core/services/helper/main.py:110]: I see!

**Pull request merged**